### PR TITLE
issue-1519: default disable a11y on touch devices

### LINKED ISSUE #1519: disabled accessibility on mobile as default
causes issues on ios more times than not. courses with accessibility on mobile require specific attention for the moment.

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #1519

### CODE REVIEW

**Comment by andrewhosgood** [comment]
Why not reverse the whole logic? Rather than `_isDisabledOnTouchDevices (default true)`, perhaps `_isEnabledOnTouchDevices (default false)`.

```js
if (!Modernizr.touch || this.isActive() || Adapt.config.get("_accessibility")._isEnabledOnTouchDevices !== true) return;
```

**Comment by oliverfoster** [comment]
That's fine if we can isolate which schemas need changing for the AT.

**Comment by oliverfoster** [comment]
@brian-learningpool @taylortom @tomgreenfield  any ideas?

**Comment by dancgray** [comment]
@oliverfoster it's in the [plugins/content/config/model.schema](https://github.com/adaptlearning/adapt_authoring/blob/master/plugins/content/config/model.schema#L57)

**Comment by oliverfoster** [comment]
much obliged sir

**Comment by oliverfoster** [comment]
just need an AT issue + PR.

**Comment by tomgreenfield** [comment]
How to do all this without a breaking change...

**Comment by oliverfoster** [comment]
what?

File: src/core/js/accessibility.js [hunk 1]
```diff
@@ -273,8 +273,12 @@ define([
                  //Remove button
                 this.$accessibilityToggle.remove();
             }
-
-            if (!Modernizr.touch || this.isActive() || Adapt.config.get("_accessibility")._isDisabledOnTouchDevices) return;
+            
+            var config = Adapt.config.get("_accessibility");
+            // Backwards compatibility for _isDisabledOnTouchDevices
+            var isEnabledOnTouchDevices = config._isEnabledOnTouchDevices || (config._isDisabledOnTouchDevices === false);
```
Comment (NayanKhedkar) [inline src/core/js/accessibility.js]: || (config._isDisabledOnTouchDevices === false);  is there any need to have this ??

Comment (oliverfoster) [inline src/core/js/accessibility.js]: Old courses which don't support _isEnabledOnTouchDevices but which do have _isDisabledOnTouchDevices set to false will still work as expected.

**Pull request merged**